README.md

```diff
@@ -92,32 +93,34 @@ Fig. 2).
 
 ### Why this Guide?
 
-We have created this Guide to help everyone get a fair shake, understand their decisions,
-and save time and effort.
-Although it is far from perfect and [still improving](#please-help), a significant number of
-experts, editors, and authors have worked to assemble and review it.
+The authors of this Guide created it to help everyone get a fair shake, understand their
+decisions, and save time and effort.
+Although it is far from perfect and still improving, we believe it is the most detailed
+overview of the subject available.
+A significant number of experts, editors, and authors have worked to assemble and review
+it.
 
-We believe you shouldn’t need a law degree or MBA to understand how you are paid.
-You shouldn’t have to know a lawyer or tax professional to orient yourself on the subject.
+You shouldn’t need a law degree or MBA to understand how you are paid.
+You shouldn’t have to blindly trust a lawyer or tax professional.
 And you shouldn’t have to fail before you succeed.
 
 While [a lot of information](#further-reading) on this topic is just a Google search away,
 the best resources are scattered about.
 Many blogs and articles focus only on a narrow topic, are getting older, or are on sites
 filled with ads or marketing copy.
-In contrast, this Guide is written to be:
-
-- **Reliable**: We do not promote any investor, company, or product.
-- **Clear**: We keep things brief and precise, but don’t oversimplify.
-- **Practical**: We include concrete details, advice, and pitfalls.
-- **Thoughtful**: We try to give context around the dry topics.
-  Some issues are also controversial or just confusing, and we work to cover points of view
-  from multiple experts when they differ.
-- **Living**: We will keep updating this material, and [welcome your contributions](#please-help)!
-  This work benefits from your suggested improvements, requests for clarification or further
-  information, and any and all questions.
-
-An early edition of this Guide has received significant discussion and feedback
+In contrast, the goals of this guide are to be **reliable** (written and reviewed by
+credible experts, and not written to promote any company or product), **practical**
+(including concrete details, advice, and pitfalls), **clear** (brief and precise), and
+**thoughtful** (giving context and multiple expert points of view, including on controversial
+topics).
+
+Also, this Guide is **living**: We continue to update this material, and
+[welcome your contributions](#please-help)! This work benefits from your suggested
+improvements, requests for clarification or further information, and any and all
+questions.
```

Comment:
Curious abt the restructure here? (Likely this will be regulated across Guides, perhaps set visually somehow.)

Reply:
Overall want to tighten up all the banter/discussion so it takes up less room. Just highlight the key points. Styling and consistency across Guides can come after, for sure.